tests/transports-integrations/tests/utils/common.py

```
@@ -0,0 +1,416 @@
+"""
+Common utilities and test data for all integration tests.
+This module contains shared functions, test data, and assertions
+that can be used across all integration-specific test files.
+"""
+
+import base64
```

Ruff (0.11.9)

`base64` imported but unused

Remove unused import: `base64`

(F401)